test/companion_web/controllers/auth_controller_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule CompanionWeb.AuthControllerTest do
+  use CompanionWeb.ConnCase
+
+  @ueberauth_auth %{
+    credentials: %{token: "aifoejwaofjaowj34"},
```

Comment:
Does the content (or format) of this token matter? If not, I'd suggest using something more obviously fake -- perhaps `pretendthisisavalidtoken` or somesuch.